lib/modulr/client.rb

```diff
@@ -57,7 +57,9 @@ def execute(method, path, data = nil, options = {})
 
     def request(method, path, data = nil, options = {})
       request_options = request_options(method, path, data, options)
+      query_params = URI.encode_www_form(options)
       uri = "#{base_url}#{path}"
+      uri = "#{uri}?#{query_params}" unless query_params.empty?
```

Comment:
This I have corrected already by using the features already built-in in Faraday, without having to encode the params manually:

![image](https://user-images.githubusercontent.com/4295/218048856-1a0962ea-7c17-4d41-ac12-124b77e1f27d.png)

It will be available in 0.0.7.